docs/multipass.md

```diff
@@ -0,0 +1,52 @@
+# Testing LibreTime with Canonical's Multipass
+
+Ever wanted to test out LibreTime but didn't want to tango with
+Virturalbox, Vagrent, or Docker?
+
+![](https://thumbs.gfycat.com/HauntingDirtyDragon-size_restricted.gif)
```

Comment:
What is this licensed under? Are we legally allowed to redistribute it?

Reply:
No idea

Reply:
Then I think it is probably best not to include it